packages/plugins/integration-make-com/src/lib/make-com-authorization.controller.ts

```
@@ -0,0 +1,116 @@
+import { Controller, Get, Query, Res, HttpException, HttpStatus } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { ApiTags, ApiOperation, ApiResponse } from '@nestjs/swagger';
+import { Response } from 'express';
+import { Public } from '@gauzy/common';
+import { IntegrationEnum } from '@gauzy/contracts';
+import { buildQueryString } from '@gauzy/utils';
+import { MakeComOAuthService } from './make-com-oauth.service';
+import { randomBytes } from 'crypto';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)